tsconfig.json

```diff
@@ -1,34 +1,28 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
+    "types": [
+      "vitest/globals",
+      "vite/client",
+      "@testing-library/jest-dom",
+      "vite-plugin-svgr/client"
     ],
-    "typeRoots": [
-      "./src/types",
-      "./node_modules/@types"
-    ],
-    "allowJs": true,
+    "typeRoots": ["./src/types"],
+    "target": "es2022",
```

Comment:
Should we have target es5 as the support for es2022 will be less I think

Reply:
We should target `ESNext `

Reply:
I have updated the config.